src/main/java/hw6/TestData.java

```diff
@@ -0,0 +1,9 @@
+package hw6;
+
+import hw6.entities.User;
+
+public class TestData {
```

Comment:
What kind of test data is that ? 
You should named class in according to it purpose.

Reply:
was renamed